pkg/tlsutil/tlsutil.go

```diff
@@ -17,6 +17,17 @@ const (
 	Duration365d = time.Hour * 24 * 365
 )
 
+type CertConfig struct {
+	CommonName   string
+	Organization []string
+	AltNames     AltNames
+}
+
+type AltNames struct {
```

Comment:
If you were thinking of the cli flags `IP=x.x.x.x,DNS=foo.com` those should be parsed prior to using this struct. These values are passed directly to the x509.Certificate object